manifests/istio-control/istio-discovery/templates/telemetryv2_1.4.yaml

```diff
@@ -15,7 +15,7 @@ spec:
       match:
         context: ANY # inbound, outbound, and gateway
         proxy:
-          proxyVersion: '1\.4.*'
+          proxyVersion: '1.4.*'
```

Comment:
won't this match `1b4`?

Reply:
no. See the code change. It is not parsed as regex at all.